### ISSUE TITLE
Cordova plugin add?

### ISSUE THREAD

**Issue by Fibinacci** [gali8/Tesseract-OCR-iOS#178]
How can add this framework add cordova hybrid project? (As the plugin)

**Comment by kevincon** [comment]
I think this is more of a question for the Cordova folks since this library was not built with that use case in mind, so can you ask them? According to their website (https://cordova.apache.org/#mailing-list) there is a developer mailing list you can join to ask questions, or you can post a question on Stack Overflow using the cordova tag.

**Comment by ws233** [comment]
@Fibinacci, have you found the answer?
Can you share your experience with us, so the opensource comunity could use your experience?

**Comment by Fibinacci** [comment]
Yes I wrote a plugin now. Basically, it reads image file and sending results back. The first version of the framework(gali8) I used did not support iOS6. Then I try to use the latest version, I had a problem while integrating into the plugin, but I figured the end. 
I have some minor problems. For example, I do not use the page segmentation feature. When I try to add any kind of, I do not get the results back(empty).

**Comment by ws233** [comment]
@Fibinacci, any updates here? have you finally succeed? Can you share your experience and code snippets here?

**Comment by okwei** [comment]
Anyone looking into this or any experience doing so?

**Comment by jcesarmobile** [comment]
I created this plugin yesterday, but it's still a beta and not ready to use yet

https://github.com/jcesarmobile/cordova-plugin-tesseract-ocr
